experimental/hip/tests/stablehlo_ops/CMakeLists.txt

```diff
@@ -5,14 +5,14 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 unset(IREE_HIP_TEST_COMPILER_FLAGS)
-if(IREE_ROCM_LINK_BC)
+if(IREE_HIP_DRIVER_LINK_BC)
```

Comment:
Thanks for decoupling the options here... this should resolve some of my comments on https://github.com/openxla/iree/pull/16706#discussion_r1518439879 too